packages/logger/src/logger.js

```diff
@@ -46,6 +46,15 @@ export class PercyLogger {
     return this.level;
   }
 
+  // get all log messages stored for the session
+  messageDetail(message) {
```

Comment:
why are we adding this ? we already have `query` function for the same. lets not repeat unless required.